docs/guides/getting_started/samples/intro/structure.cs

```diff
@@ -36,7 +36,7 @@ private Program()
             // you must set the MessageCacheSize. You may adjust the number as needed.
             //MessageCacheSize = 50,
 
-            // If your platform doesn't have native websockets,
+            // If your platform doesn't have native WebSockets,
```

Comment:
IMO there was no need for this change

Reply:
I'm actually not sure how that got there; I didn't explicitly make that change in my IDE 🤔